scripts/mock_openai.py

```
@@ -281,14 +283,34 @@ def tc_multi(pairs):
+        # absolute path (a temp dir on the host) and declares it via
+        # authorize_paths — the desktop must pop the write-authorization card
+        # (允许本次 / 本目录不再询问 / 拒绝) instead of silently writing.
         last_user = next(
             (str(m.get("content", "")) for m in reversed(messages) if m.get("role") == "user"),
             "",
         )
+        if "写授权" in last_user:
+            out_dir = os.environ.get("MIQI_AUTH_OUT_DIR", "")
+            target = os.path.join(out_dir, "auth_probe.txt") if out_dir else ""
+            if not target:
+                self._respond(text("写授权：缺少 MIQI_AUTH_OUT_DIR 环境变量。"))
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)